Makefile

```diff
@@ -336,3 +336,11 @@ format-drone:
 
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+rebuild-pmm:
+		make build-go
+		rm -f /usr/sbin/grafana-server
+		rm -f /usr/sbin/grafana
+		cp ./bin/linux-amd64/grafana-server /usr/sbin/grafana-server
```

Comment:
Both `grafana-server` and `grafana-cli` are not used, so we can remove this line.

Reply:
removed all mentions of grafana-server and grafana-cli